pkgs/games/gcs/default.nix

```diff
@@ -1,78 +1,75 @@
-{ lib, stdenv, fetchFromGitHub, runCommand
-, jdk8, ant
-, jre8, makeWrapper
+{ lib
+, buildGoModule
+, pkg-config
+, libX11
+, libXcursor
+, libXrandr
+, libXinerama
+, libXi
+, libXxf86vm
+, libGL
+, freetype
+, fontconfig
+, zlib
+, harfbuzz
+, openjpeg
+, jbig2dec
+, libjpeg_turbo
+, gumbo
+, mupdf
+, fetchFromGitHub
 }:
 
 let
+  ver = "5.15.0";
```

Comment:
Why not move this and the gcs into the buidGoPackage?

Reply:
No reason not to i just had it in the let because the previous derivation had it inside the let instead of inside the call to mkDerivation.

Reply:
do you want me top move it there?

Reply:
Yes, remove this `ver`.

Usually the `let . . . in` is employed in order to avoid `rec`. But the expression is using `rec` already, making this `let` useless.